packages/editor-ui/src/views/SettingsUsageAndPlan.vue

```diff
@@ -53,34 +53,36 @@ const onLicenseActivation = async () => {
 };
 
 onMounted(async () => {
-	usageStore.setLoading(true);
-	if (route.query.key) {
+	if (!usageStore.isDesktop) {
```

Comment:
Nit - inverting the check and returning early would prevent the nesting and diff.